tests/cli_test.py

```diff
@@ -0,0 +1,51 @@
+from datetime import datetime, timedelta
+
+import pytest
+from flask import current_app
+
+from newdle.core.db import db
+from newdle.models import Newdle
+
+
+@pytest.mark.usefixtures('db_session')
+def test_cleanup_newdles_uncompleted_tasks(cli_runner, dummy_newdle):
+    dummy_newdle.last_update = datetime.utcnow() - timedelta(
+        days=current_app.config['LAST_ACTIVITY_CLEANUP_INTERVAL'] + 1
+    )
+    db.session.commit()
+    assert 1 == Newdle.query.count()
+    cli_runner.invoke(args=['cleanup_newdles'])
+    assert 0 == Newdle.query.count()
+
+
+@pytest.mark.usefixtures('db_session')
+def test_cleanup_newdles_non_fitting_incomplete_tasks(cli_runner, dummy_newdle):
+    dummy_newdle.last_update = datetime.utcnow() - timedelta(
+        days=current_app.config['LAST_ACTIVITY_CLEANUP_INTERVAL'] - 10
+    )
+    db.session.commit()
+    assert 1 == Newdle.query.count()
+    cli_runner.invoke(args=['cleanup_newdles'])
+    assert 1 == Newdle.query.count()
+
+
+@pytest.mark.usefixtures('db_session')
+def test_cleanup_newdles_final_tasks(cli_runner, dummy_newdle):
+    dummy_newdle.final_dt = datetime.utcnow() - timedelta(
+        days=current_app.config['FINAL_DATE_CLEANUP_INTERVAL'] + 1
+    )
+    db.session.commit()
+    assert 1 == Newdle.query.count()
+    cli_runner.invoke(args=['cleanup_newdles'])
+    assert 0 == Newdle.query.count()
+
+
+@pytest.mark.usefixtures('db_session')
+def test_cleanup_newdles_non_fitting_final_tasks(cli_runner, dummy_newdle):
+    dummy_newdle.final_dt = datetime.utcnow() - timedelta(
+        days=current_app.config['FINAL_DATE_CLEANUP_INTERVAL'] - 5
```

Comment:
If black lets you do it, I'd prefer this since it looks a bit weird when the value of a kwarg has spaces and there aren't parentheses around it:

```suggestion
        days=(current_app.config['FINAL_DATE_CLEANUP_INTERVAL'] - 5)
```